src/datepicker/datepicker.component.spec.ts

```diff
@@ -0,0 +1,119 @@
+import { TestBed } from "@angular/core/testing";
+import { By, EVENT_MANAGER_PLUGINS } from "@angular/platform-browser";
+import { Component, Output, EventEmitter } from "@angular/core";
+import { DatePicker } from "./datepicker.component";
+import { DatePickerInput } from "../datepicker-input/datepicker-input.component";
+import { Calendar16Module } from "@carbon/icons-angular/lib/calendar/16";
+import { FormsModule } from "@angular/forms";
+
+@Component({
+	template: `
+	<ibm-date-picker
+		[label]="label"
+		[placeholder]="placeholder"
+		[theme]="theme"
+		[value]="value"
+		[disabled]="disabled"
+		[invalid]="invalid"
+		[invalidText]="invalidText"
+		[dateFormat]="dateFormat"
+		(valueChange)="valueChange($event)">
+	</ibm-date-picker>
+	`
+})
+class DatePickerTest {
+	label = "Date picker label";
```

Comment:
Some of these properties can be set inline in the template since they are never changed.